crdppf/templates/derived/init.js

```diff
@@ -78,12 +78,19 @@ Crdppf.baseLayersList = {'baseLayers': [
         %>
     % endfor
 ]};
-Crdppf.defaultTiles = {}
-% if len(baseLayers) > 1:
-    Crdppf.defaultTiles = {
-        'wmtsname' : "${baseLayers[0]['wmtsname']}",
-        'tile_format' : "${baseLayers[0]['tile_format']}"
-    }
+
+% if len(baseLayers) > 0:
+    Crdppf.defaultTiles = Crdppf.baseLayersList['baseLayers'][0];
+% else:
+    % if 'defaultTiles' in request.registry.settings.keys() :
+        Crdppf.defaultTiles = {${request.registry.settings['defaultTiles']|n}};
```

Comment:
You have to indent javascript and mako independently, thus, here, you javascript code is too indented